examples/app.R

```diff
@@ -5,7 +5,7 @@ VERSION = "1.2.2"
 
 shinyApp(
   ui = tagList(
-    shiny.info::powered_by("Appsilon", "https://appsilon.com", "bottom right"),
+    shiny.info::busy(position="top left"),
```

Comment:
I would leave it as powered_by or git_info in the example. Busy hides fast after the app is loaded and it may make users think that nothing is displayed.

Reply:
True!